### PR TITLE
Add collection information to NFTs

### PR DESCRIPTION
**Pull request by yabirgb**
it adds a new object `collection` to the response data with:

- `name`
- `description`
- `large_image`
- `banner_image`

### CODE REVIEW

File: rotkehlchen/tests/api/test_nfts.py [hunk 1]
```diff
@@ -54,4 +54,5 @@ def test_nft_query(rotkehlchen_api_server, start_with_valid_premium):
             assert entry['image_url'] == 'https://lh3.googleusercontent.com/kwF-39qZlluEalQnNv-yMxbntzNdc3g00pK2xALkpoir9ooWttVUO2hVFWOgPtOkJOHufYRajfn-nNFdjruRQ4YaMgOYHEB8E4CdjBk'  # noqa: E501
             assert FVal(entry['price_eth']) > ZERO
             assert FVal(entry['price_usd']) > ZERO
+            assert entry['collection']['name'] == 'BASTARD GAN PUNKS V2'
```
Comment (LefterisJP) [inline rotkehlchen/tests/api/test_nfts.py:57]: Cant you also check for some other parts of the collection object?

**Review by LefterisJP — APPROVED** [review]
LGTM without having tested but just looking at the api

**Comment by yabirgb** [comment]
Failing tests is for aave having a 500 error from etherscan https://github.com/rotki/rotki/pull/3409/checks?check_run_id=3512762416#step:8:527

**Review by LefterisJP — APPROVED** [review]
LGTM. Thank you ser.

**Pull request merged**